.github/workflows/release.yml

```diff
@@ -91,6 +91,19 @@ jobs:
           name: windows-binary
           path: dist/newrelic_windows_amd64_v1/newrelic.exe
 
+      - name: Sign newrelic.exe
+        shell: bash
+        env:
+          PFX_CONTENT: ${{ secrets.PFX_BASE64_CONTENT }}
+          PFX_PASSWORD: ${{ secrets.PFX_CERT_PASSWORD }}
+        run: |
+          echo "$PFX_CONTENT" | base64 -d > cert.pfx
+          sudo apt-get install osslsigncode -y
+          sudo mv dist/newrelic_windows_amd64_v1/newrelic.exe dist/newrelic_windows_amd64_v1/newrelic-unsigned.exe
+          osslsigncode sign -pkcs12 cert.pfx -pass "$PFX_PASSWORD" -h sha512 -t http://timestamp.digicert.com \
+            -in dist/newrelic_windows_amd64_v1/newrelic-unsigned.exe -out dist/newrelic_windows_amd64_v1/newrelic.exe
```

Comment:
@amolero-nr I think this looks good. I might also suggest to remove the file once signing is complete newrelic-unsigned.exe

Reply:
Oh true! Thanks!